### ISSUE TITLE
Features request: autologin

### ISSUE THREAD

**Issue by wfx** [nwg-piotr/nwg-hello#21]
Yes, enabling automatic login poses security risks, but i in my situation I like it (using dm-crypt).
It would be an optional feature :)

thx

**Comment by nwg-piotr** [comment]
I've never tried, but it seems to be the daemon (greetd) feature, not the greeter (nwg-hello) itself. 

https://wiki.archlinux.org/title/Greetd

**Comment by wfx** [comment]
Yes, you have right, is it possible to add a checkbox in nwg-hello to enable/disable it?

**Comment by nwg-piotr** [comment]
I don't really know at the moment. It would require edition of the greeter config file and some experimenting. Will take a look at it when I have several spare hours.
